src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleMap = sourceString
+    .split('\n')
+    .map(el => el.trim())
+    .filter(el =>
+      el !== '' && el !== ';').map(el => (
+      el.split(':').map(item => item.trim())
+    ));
```

Comment:
```suggestion
    .filter(
      el => el !== '' && el !== ';'
    ).map(
      el => el.split(':').map(item => item.trim()
    ));
```
Учи форматирование, это очень важно. После стрелки строку не переносим, а переносим ее вместе со всем остальным, до параметра коллбека. Потом, каждый метод пишем с новой строки, а вот его начинку, по возможности, оставляем в одной строке